app/src/main/java/com/android/voyageur/model/user/UserViewModel.kt

```diff
@@ -240,4 +245,13 @@ open class UserViewModel(
           }
         }
   }
+
+  fun getUsersByIds(userIds: List<String>, onSuccess: (List<User>) -> Unit) {
```

Comment:
Documentation is essential especially in the ViewModel, that way other members can quickly understand what a specific method does.